### PR TITLE
W-14221349 Pdk troubleshooting init

### PR DESCRIPTION
**Pull request by NachoRaik**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: gateway/1.5/modules/ROOT/pages/policies-pdk-compile-policies.adoc [hunk 1]
```diff
@@ -35,3 +35,7 @@ The `build` command preforms the following actions:
 The `build` command outputs the generated files to the `target` directories.
 
 You can execute the `build` command as many times as necessary to generate compiled artifacts. You must compile the custom policy for your source code and configuration edits to be present in the compiled artifacts.
+
+== See Also
+
+* policies-pdk-troubleshooting.adoc[]
```
Comment (hannanelson) [inline gateway/1.5/modules/ROOT/pages/policies-pdk-compile-policies.adoc]: Does this build correctly? It looks like it needs the `xref:` bit.

File: gateway/1.5/modules/ROOT/pages/policies-pdk-configure-features.adoc [hunk 2]
```diff
@@ -585,3 +585,7 @@ It is possible that some libraries compile properly to the `wasm32-wasi` target
 * `Plugin configured to fail closed failed to load`
 
 In this case, contact the owners of the third-party library or use a different library.
+
+== See Also
+
+* policies-pdk-troubleshooting.adoc[]
```
Comment (hannanelson) [inline gateway/1.5/modules/ROOT/pages/policies-pdk-configure-features.adoc]: `xref:`

File: gateway/1.5/modules/ROOT/pages/policies-pdk-create-project.adoc [hunk 3]
```diff
@@ -139,3 +139,7 @@ For more information about publishing policies in Exchange, see xref:policies-pd
 * `make release`: Releases the policy asset in Exchange. 
 +
 For more information about releasing policies in Exchange, see xref:policies-pdk-publish-policies.adoc[].
+
+== See Also
+
+* policies-pdk-troubleshooting.adoc[]
```
Comment (hannanelson) [inline gateway/1.5/modules/ROOT/pages/policies-pdk-create-project.adoc]: `xref:`

File: gateway/1.5/modules/ROOT/pages/policies-pdk-create-schema-definition.adoc [hunk 4]
```diff
@@ -363,4 +363,8 @@ For example, you can access the values in `struct Config` with the following `.o
 async fn configure(launcher: Launcher, Configuration(configuration): Configuration,) -> Result<(), LaunchError> { 
     let config: Config = serde_json::from_slice(&configuration)?; 
     match config.reject_missing_tokens {
-----
\ No newline at end of file
+----
+
+== See Also
+
+* policies-pdk-troubleshooting.adoc[]
```
Comment (hannanelson) [inline gateway/1.5/modules/ROOT/pages/policies-pdk-create-schema-definition.adoc]: `xref:`

File: gateway/1.5/modules/ROOT/pages/policies-pdk-debug-deployed-policies.adoc [hunk 5]
```diff
@@ -17,7 +17,7 @@ The PDK debugging environment provides an easy way to view Flex Gateway logging
 For more information about PDK's debugging environment, see xref:policies-pdk-debug-local.adoc[].
 
 [[debug-logs]]
-== Troubleshoot in a Production Environment
+== Debug in a Production Environment
 
 It is possible that errors that are not present in PDK's debugging environment are present when the policy is deployed to a production environment.
```
Comment (hannanelson) [inline gateway/1.5/modules/ROOT/pages/policies-pdk-debug-deployed-policies.adoc]:
```suggestion
Errors that are not present in the PDK debugging environment might be present when the policy is deployed to a production environment.
```

Comment (hannanelson) [reply gateway/1.5/modules/ROOT/pages/policies-pdk-debug-deployed-policies.adoc]: Don't use possessives with product/feature names. 

Is PDK an approved initialism?

File: gateway/1.5/modules/ROOT/pages/policies-pdk-debug-local.adoc [hunk 6]
```diff
@@ -159,3 +159,7 @@ To learn more about editing a Flex Gateway's logging configuration file, see xre
 Visual Studio Code (VS Code) includes a Rust debugger that can run unit tests. You cannot use the VS Code Debugger while the policy is deployed on a Flex Gateway. To create the API mocking, you must use unit tests for the specific logic you need to debug.
 
 To debug Rust code in VS Code, you must install the https://marketplace.visualstudio.com/items?itemName=vadimcn.vscode-lldb[CodeLLDB Debugger Extension^] or a different VS Code extension with the same functionality. To learn more about debugging Rust in VS Code, see https://code.visualstudio.com/docs/languages/rust#_debugging[Rust in Visual Studio Code^].
+
+== See Also
+
+* policies-pdk-troubleshooting.adoc[]
```
Comment (hannanelson) [inline gateway/1.5/modules/ROOT/pages/policies-pdk-debug-local.adoc]: `xref:`

File: gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc [hunk 7]
```diff
@@ -80,3 +80,7 @@ For example, if the latest version is `1.0.0`, update the `Cargo.toml` file as f
 name = "my-custom-policy"
 version = "1.1.0"
 ----
+
+== See Also
+
+* policies-pdk-troubleshooting.adoc[]
```
Comment (hannanelson) [inline gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc]:
```suggestion
* xref:policies-pdk-troubleshooting.adoc[]
```

File: gateway/1.5/modules/ROOT/pages/policies-pdk-troubleshooting.adoc [hunk 8]
```diff
@@ -0,0 +1,58 @@
+= Policy Development Kit Troubleshooting
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+When using the Policy Development Kit (PDK), you might encounter issues with PDK itself. See the following sections to help you resolve these issues:
```
Comment (hannanelson) [inline gateway/1.5/modules/ROOT/pages/policies-pdk-troubleshooting.adoc]: The intro implies multiple sections, but only one bullet appears.

File: gateway/1.5/modules/ROOT/pages/policies-pdk-troubleshooting.adoc [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+
+* xref:policies-pdk-debug-local.adoc[]
+* xref:policies-pdk-debug-deployed-policies.adoc[]
+
+
+== Cargo Credential-Provider Configuration
+
+When running the `setup` or `build` commands, you might receive the following error:
+
+----
```
Comment (hannanelson) [inline gateway/1.5/modules/ROOT/pages/policies-pdk-troubleshooting.adoc:22]: Should this include any syntax highlighting? `[source,???]`

File: gateway/1.5/modules/ROOT/pages/policies-pdk-troubleshooting.adoc (hunk 8 above)
Comment (glenn-rodgers-sf) [reply gateway/1.5/modules/ROOT/pages/policies-pdk-troubleshooting.adoc]: I'm kind of assuming that there will be more sections eventually, but I could remove this for now?

Comment (hannanelson) [reply gateway/1.5/modules/ROOT/pages/policies-pdk-troubleshooting.adoc]: If it says sections, users will wonder if there's something missing. You can change it to section for now?

**Review by hannanelson — APPROVED** [review]

**Review by hannanelson — APPROVED** [review]

**Review by hannanelson — APPROVED** [review]

**Pull request merged**